docs/getting_started.md

```diff
@@ -473,29 +473,29 @@ The API provides the macro `PSA_HASH_SIZE`, which returns the expected hash leng
 
 #### Handling hash operation contexts
 
-Once the operation structure has been successfully initialized by a successful call to `psa_hash_setup`, it's possible to terminate the operation at any time by calling `psa_hash_abort`. The call to `psa_hash_abort` frees any resources associated with the operation (except for the operation structure itself).
+After a successful call to `psa_hash_setup()` initializes the operation structure, you can terminate the operation at any time by calling `psa_hash_abort()`. The call to `psa_hash_abort()` frees any resources associated with the operation, except for the operation structure itself.
```

Comment:
Preexisting: please remove “initializes the operation structure”. `psa_hash_setup` does not actually initialize the operation structure. “Initialize” and “setup” are different steps, but `psa_hash_setup` did both in the past and we hadn't updated the wording here.

Reply:
This needs updating still.